internal/rpc/models.go

```diff
@@ -117,28 +115,15 @@ var validateFormRPC = starlark.MakeStarlarkFunction(
     for name in properties:
         if name not in data:
             continue
-        x = FindOne("datatype", Eq("name", properties[name]["datatype"]))
-        if not x.Get("enum"):
-            y = FindOne("code", EqID(x.GetFK("validator")))
+        x = aft.findOne("datatype", {"where" : {"name" : properties[name]["datatype"]}, "include" : {"validator" : True}})
```

Comment:
this is broken in a few ways, but no worries if you want to punt on it

Reply:
Gonna punt, because I think I should be using cases. Importantly the lint and terminal work. I'll get to the otherones later

Reply:
sg!